services/minespace-web/src/components/dashboard/mine/reports/Reports.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { FC, useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PlusCircleFilled from "@ant-design/icons/PlusCircleFilled";
+import { Button, Col, Row, Typography } from "antd";
+import moment from "moment";
+import PropTypes from "prop-types";
```

Comment:
wait, are you still using the PropTypes import? CustomPropTypes? I'm seeing a lot of unused imports lately... We should add an eslint rule and make it an error (instead of a warning).

Reply:
Good catch!  Using Intellij I usually just run `shift + cmd + l` and it runs a "clean up imports" function.  Forgot to run it on this one.  But I agree a rule to catch unused imports would be useful.  I'll look into it.